tests/integration/main.rs

```diff
@@ -0,0 +1,4 @@
+mod common;
+mod os;
+mod pie;
+mod sharp;
```

Comment:
This is just a dummy main? I imagine there's a cleaner way to prevent warnings...